lib/ts/recipe/passwordless/components/themes/translations.ts

```diff
@@ -54,6 +54,10 @@ export const defaultTranslationsPasswordless = {
         PWLESS_SIGN_IN_UP_CHANGE_CONTACT_INFO_EMAIL: "Change email",
         PWLESS_SIGN_IN_UP_CHANGE_CONTACT_INFO_PHONE: "Change phone number",
 
+        PWLESS_LINK_CLICKED_CONTINUE_HEADER: "Sign Up or Log In",
```

Comment:
I don't think translation segments should be extracted into constants inside these files, it'd make reading them harder if someone wants to override them, and we really gain much by it.
Reusing the other key is not really that great either because this is a separate screen with entirely separate customizability. I was thinking about making this a bit more specific because it seems disconnected from the other screens where this is the header.

Reply:
Ok. Let's not use variable and stick to what was done already.